packages/circuit-ui/components/RadioButton/RadioButton.tsx

```diff
@@ -255,11 +273,23 @@ export const RadioButton = forwardRef(
         />
         <RadioButtonLabel
           htmlFor={id}
-          invalid={invalid}
+          invalid={invalid ?? false}
           className={className}
           style={style}
+          aria-describedby={helperText ? `${id}-helper` : undefined}
         >
           {label}
+          {helperText && (
+            <>
+              <br />
+              <RadioButtonHelperText
+                id={`${id}-helper`}
+                data-testid={`${label}-helper`}
```

Comment:
We avoid test-only attributes in Circuit UI. Instead, query the element by its role or text content.